src/sdi-notify.c

```diff
@@ -192,7 +192,8 @@ static void show_pending_update_notification(SdiNotify *self,
                                  _("Show updates"), app_show_updates,
                                  g_object_ref(self), g_object_unref);
   notify_notification_add_action(notification, "default", _("Close"),
-                                 app_close_notification, NULL, NULL);
+                                 app_show_updates, g_object_ref(self),
+                                 g_object_unref);
```

Comment:
It seems quite confusing that both "Show updates" and "Close" do the same thing - It might be worth adding a comment here.

Reply:
Done.